### PR TITLE
PIM-8463: Explicit error message for import profile max file size

### PR DESCRIPTION
**Pull request by jmleroux**
When a user try to import a big file into for an import profile, we don't check the file size and we try to upload it even if it's too large to be accepted.

This PR exposes the server upload_max_filesize in the JS user context to be able to validate the file size before trying to upload it.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/UIBundle/Controller/InternalApi/EnvironmentController.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            [
+                'upload_max_file_size' => $this->serverMaxFileSize(),
+            ]
+        );
+    }
+
+    /**
+     * @return int Upload max filesize in Mb
+     */
+    private function serverMaxFileSize(): int
```
Comment (juliensnz) [inline src/Akeneo/Platform/Bundle/UIBundle/Controller/InternalApi/EnvironmentController.php]: it should be exposed as bytes, the presentational part should be done in front

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/common/environment.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        };
+
+        return {
+            /**
+             * Returns server variables
+             *
+             * @return {Promise}
+             */
+            getVariables: function () {
+                return loadEnvironment().then(config => config);
```
Comment (juliensnz) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/common/environment.js]: I don't get what config => config does

**Comment by jmleroux** [comment]
> Then expose it only in the requirejs or form extension (not doing a fetch) or add it to an already existing context (user-context for example) to avoid a new fetch

good idea !

**Comment by jmleroux** [comment]
> I don't think it's the good approach: this should be a parameter in the configuration of the PIM (system section) and not linked to the php settings.

The purpose is to be fully compatible with cloud settings. I will see with the Cloud team how they configure this limit. But i don't think they want to use php or yaml file because it's very environment oriented.

**Comment by jmleroux** [comment]
Does not solve the right issue

**Pull request closed without merging**